Makefile

```diff
@@ -24,6 +26,15 @@ build-tools:
 	cd third_party/yosys && PREFIX=${INSTALL_DIR} $(MAKE) -j`nproc` && PREFIX=${INSTALL_DIR} $(MAKE) install
 	# Build Yosys plugins
 	cd third_party/yosys-plugins && export PATH=${INSTALL_DIR}/bin:${PATH} && $(MAKE)
```

Comment:
The `build-tools` target should probably be split into multiple `build-tools-XXX` targets.

Reply:
Done, each tool is under `build-<toolname>`